docs/reference/reference-training/training/api-pytorch-trainer-reference.rst

```diff
@@ -0,0 +1,120 @@
+###############################
+ PyTorch Trainer API Reference
+###############################
+
++----------------------------------------------------+
+| User Guide                                         |
++====================================================+
+| :doc:`/training/apis-howto/api-pytorch-trainer-ug` |
++----------------------------------------------------+
+
+.. code::
+
+   determined.pytorch.init(
+       *,
+       hparams: Optional[Dict] = None,
+       exp_conf: Optional[Dict[str, Any]] = None,
+       distributed: Optional[core.DistributedContext] = None,
+       aggregation_frequency: int = 1,
+   ) -> pytorch.PyTorchTrialContext:
+
+``pytorch.init()`` builds a ``pytorch.PyTorchTrialContext`` for use with ``PyTorchTrial``.
```

Comment:
These should be autodoc references so they form hyperlinks.

This pr comment applies throughout the PR, though not every single reference needs to be a link.  I do think at least the first time a particular type is discussed in a section should always link to the object I think, and after that using the double-backticks for plain monospaced text seems fine.  Kind of like nouns and pronouns, you can get away with using pronouns if the last use of a noun is sufficiently recent.